var/spack/repos/builtin/packages/py-shiboken/package.py

```diff
@@ -18,13 +18,21 @@ class PyShiboken(PythonPackage):
 
     depends_on("cmake", type="build")
 
+    # to prevent error: 'PyTypeObject' {aka 'struct _typeobject'} has no member
+    # named 'tp_print'
+    depends_on("python@:3.8", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     # in newer pip versions --install-option does not exist
     depends_on("py-pip@:23.0", type="build")
     depends_on("py-sphinx@:3.4", type=("build", "run"))
     depends_on("libxml2")
     depends_on("qt@:4.8")
```

Comment:
```suggestion
    depends_on("qt@4.6:4.8")
```
Based on setup.py. Then you no longer need a conflict.

Reply:
I didn't see that restriction, thanks.